src/user.ts

```diff
@@ -6,6 +6,16 @@ import type {
   MarqetaError,
 } from './'
 
+export interface Transition {
+  idempotentHash?: string;
+  token?:	string;
+  status:	string;
+  reasonCode:	string;
+  reason?:string;
+  channel:	string;
+  userToken: string;
+}
```

Comment:
OK... I'll bite... what's the deal with the odd indentation? Why didn't Lint freak out about multiple spaces?

Reply:
In this `interface` definition, you have:

```typescript
 export interface Transition {
   idempotentHash?: string;
   token?:	string;
   status:	string;
   reasonCode:	string;
   reason?:string;
   channel:	string;
   userToken: string;
 }
```

but in all the others, there is only one space between the `:` of the name, and the type definition, as if it looks like:

```typescript
 export interface Transition {
   idempotentHash?: string;
   token?: string;
   status: string;
   reasonCode: string;
   reason?: string;
   channel: string;
   userToken: string;
 }
```

I think it was that you had TABs in there... and it was looking OK on your screen, but odd in the diff on GitHub...